docs/admin/troubleshooting/cluster-health-check.md

```diff
@@ -1,8 +1,82 @@
-# Verifying Cluster Health
+# Cluster Health and Troubleshooting
 
-Following is a set of tests that determine the Run:ai cluster health:
+## Cluster is Disconnected
 
-## Verify that data is sent to the cloud
+When a cluster is in the status “Disconnected”, this means that no communication from the Run:ai cluster services reaches the Run:ai Control Plane. 
+
+This may reflect a networking issue from or to your Kubernetes cluster regardless of Run:ai components. In some cases, it may indicate an issue with one or more Run:ai services that communicate with the Control Plane, which are:
+* Cluster sync (`cluster-sync`)
+* Agent (`runai-agent`)
+* Asset sync (`asset-sync`)
+
+### Troubleshooting actions:
+
+* First, check that the Run:ai services that communicate with the Control Plane are up and running.
```

Comment:
@gal-revach are these steps in a procedure? If yes, then they need to be changed to a numbered list - meaning `1.` and `2.` etc. If you want to make the suggested change let me know. If not then I will merge and make the adjustments myself.

Reply:
@jasonnovichRunAI we can do that as a procedure because they come one after the other... I just didn't find a way to make it look nice. If you have a way to do it it seems good to me